js/components/TutorialPanel.jsx

```diff
@@ -0,0 +1,59 @@
+var BootstrapModalMixin = require('../mixins/BootstrapModalMixin.jsx');
+var strings = require('../lib/strings_utils.js');
+var React = require('react');
+
+var TutorialPanel = React.createClass({
+  mixins: [BootstrapModalMixin],
+  propTypes: {
+    title: React.PropTypes.string,
+    content: React.PropTypes.string,
+    clazz: React.PropTypes.string,
+  },
+
+  getInitialState: function() {
+    return {
+      step: 0,
+    };
+  },
+
+  goToNext: function() {
```

Comment:
That's fair. I tried doing that at first, but it didn't make sense or work very well. We could remove the TutorialManager all together. Would you prefer that?